types/elasticsearch/elasticsearch-tests.ts

```diff
@@ -338,22 +338,42 @@ client.indices.updateAliases({
   // ...
 });
 
-client.reindex({
-    body: {
-        source: {
-          index: "twitter"
-        },
-        dest: {
-          index: "new_twitter"
+client.indices.updateAliases({
+  body: {
+    actions: [
+      {
+        add: {
```

Comment:
would you mind adding test for `remove` as well since we also change that part?

Reply:
Actually I rechecked the doc https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-aliases.html#deleting) and these parameters are only available for the `add` command.

So I removed the change to the `remove` one altogether.